### DISCUSSION TITLE
Import / Export-Data

### DISCUSSION THREAD

**Discussion by creating** [te-online/timemanager#42] [Ideas]
Hey, at first thank you very much for the app. It's fast, simple and fits perfect.
For sure there some improvements, which are on the roadmap, like Multi-User-Stuff.
I would directly change to your app, if I could import my data from invoiceninja. Might work directly with MySQL, but maybe this could be interesting for others as well.

Cheers
Dirk

**Comment by creating** [comment]
Und was mega hilfreich wäre, wenn Kunden -> Projekte und Co. Falls nicht vorhanden direkt mit angelegt werden. Momentan ist die UI etwas schwierig, wenn ich vergessen hab einen Kunden, dann ein Projekt und dann eine Aufgabe anzulegen.

**Reply by te-online** [reply]
Agreed, the UI around creating time entries on the Dashboard could be a lot better 🙈

**Comment by te-online** [comment]
If you can get you data from Invoiceninja in CSV or JSON, you might be able to transform it to a JSON blob that can be accepted by the REST API exposed by the app.

However, the API is currently undocumented. Take a look at this issue for some background on the REST API while it is in its undocumented state: https://github.com/te-online/timemanager/issues/8

Would that be an option for you?
